lib/common_test/src/ct_groups.erl

```diff
@@ -58,7 +58,7 @@ find_groups1(Mod, GrNames, TCs, GroupDefs) ->
 	    Path ->
 		{Path,true}
 	end,
-    TCs1 = if (is_atom(TCs) and (TCs /= all)) or is_tuple(TCs) ->
+    TCs1 = if (is_atom(TCs) andalso (TCs /= all)) orelse is_tuple(TCs) ->
```

Comment:
I think that you should remove redundant parentheses on both side of `andalso`/`orelse`.

```suggestion
    TCs1 = if (is_atom(TCs) andalso TCs /= all) orelse is_tuple(TCs) ->
```